ROLES.md

```diff
@@ -145,36 +133,35 @@ the area.
 
 Members who frequently contribute code are expected to proactively perform code reviews for the area that they are active in.
 
-## Approver
+## Maintainer
 
-Code approvers are able to both review and approve code contributions. While code review is focused on code quality and correctness,
+Maintainers review and approve code contributions. While code review is focused on code quality and correctness,
 approval is focused on holistic acceptance of a contribution including: backwards / forwards compatibility, adhering to API and flag
-conventions, subtle performance and correctness issues, interactions with other parts of the system, etc. Approver status is scoped to a
+conventions, subtle performance and correctness issues, interactions with other parts of the system, etc. Maintainer status is scoped to a
 part of the codebase.
 
 ### Requirements
 
-The following apply to the part of the codebase for which one would be an approver in an OWNERS file:
+The following apply to the part of the codebase for which one would be a maintainer:
 
 * Member for at least 3 months
 
 * Contributed at least 30 substantial PRs to the codebase
 
-* Knowledgeable about the codebase
-
-* Nominated by an area lead
+* Must remain an active participant in the community by contributing
+code, performing reviews, triaging issues, etc.
```

Comment:
For additional consideration or later addition: define emeritus status for maintainers and the faster path for re-achieving maintainer status.

Reply:
Done.